docs/web-apps/live-testing/live-cross-browser-testing.md

```diff
@@ -107,67 +107,6 @@ Know if your app is designed to run on internal or otherwise restricted networks
 | <img src={useBaseUrl('img/live-testing/clear-log-icon.png')} alt="Clear Log icon" width="40"/> | Clear Log | Clears the log feed. |
 | <img src={useBaseUrl('img/live-testing/download-log-icon.png')} alt="Download Log icon" width="40"/> | Download Log | Downloads the log as a .txt file. |
 
-### Device Vitals
```

Comment:
In the Live Mobile Real Device Test Interface table above (in https://docs.saucelabs.com/web-apps/live-testing/live-cross-browser-testing/#live-mobile-real-device-test-interface), the description for the Developer Options icon mentions the Device Vitals tab -- we should update that too.
![Screen Shot 2022-06-28 at 11 56 03 AM](https://user-images.githubusercontent.com/15352964/176226402-33c7eb58-e90e-4744-9948-cc01a5c32b52.png)

Reply:
good catch! thanks! done!